MREUnityRuntime/MREUnityRuntimeLib/Core/VideoStreamDescription.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MixedRealityExtension.Core
+{
+    public class VideoStreamDescription : UnityEngine.ScriptableObject
```

Comment:
>ScriptableObject [](start = 54, length = 16)

For my education: What is the benefit of deriving from ScriptableObject?

Reply:
Got my answer offline, thanks!

Reply:
Documenting the answer: It is because our other asset manager is using UnityEngine.Object, not System.Object, and ScriptableObject is the right way to create a non-monobehavior object derived from UnityEngine.Object - UnityEngine.Object is a custom class that requires native engine functionality for instantiation